src/game/game.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Battleships game</title>
+</head>
+<body>
+    <main class="wrapper">
+        <div class="wrapper__main">
+            <button class="startGameButton">Let's play!</button>
+            <div class="timer">
+                <span class="timer__counter">10:00</span>
+            </div>
+            <div class="board__wrapper">
+                <div class="board__player">
+                    <h1 class="board__player-name player">Player</h1>
+                </div>
+                <div class="board__board">
+                    <div class="board__board-fieldsWrapper"></div>
+                    <div class="board__charactersWrapper"></div>
+                    <div class="board__numbersWrapper"></div>
+                </div>
+                <div class="shipsWrapper">
+                    <h1 class="shipsWrapper__heading">Your ships:</h1>
+                    <div class="shipsWrapper__ships"></div>
+                    <div class="shipsWrapper__buttons">
```

Comment:
Based upon my knowledge I think it should be `ships__buttons` or even just `actions`.